src/app/pages/service/photos/photos.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http'
+import { environment } from '../../../environments/environment'
```

Comment:
The relative path is perfectly fine but I usually use absolute path whose root is src/ which I find more convenient and easy to track.
We shall discuss more on this with other devs.